.gitattributes

```diff
@@ -0,0 +1,8 @@
+# Ensure that all files have line feeds for line endings, so that they're consistent regardless of
+# which platform a change was developed on.
+* text=auto eol=lf
```

Comment:
This just means people can use the repo on Windows without inadvertently changing all the line endings. I just threw it in here for good measure, but it's not related to Yarn or workspaces.